src/test/java/org/wiremock/integrations/testcontainers/WireMockContainerExtensionTest.java

```diff
@@ -43,7 +43,11 @@ public class WireMockContainerExtensionTest {
     public WireMockContainer wiremockServer = new WireMockContainer("2.35.0")
             .withStartupTimeout(Duration.ofSeconds(60))
             .withMapping("json-body-transformer", WireMockContainerExtensionTest.class, "json-body-transformer.json")
-            .withExtension("JSON Body Transformer", Collections.singleton("com.ninecookies.wiremock.extensions.JsonBodyTransformer"),
+            .withExtension("Webhook",
```

Comment:
Maybe a separate test then?

Reply:
Make sense. Code with two extensions moved to separate test class